ru/about/index.md

```diff
@@ -39,20 +39,20 @@ Ruby-Talk, основная [рассылка](/ru/community/mailing-lists/) д
 стало использовать несколько тематических рассылок вместо одной общей.
 
 Индекс [TIOBE][tiobe], который измеряет рост популярности языков
-программирования, помещает Ruby на 9 место среди общепризнанных языков
+программирования, помещает Ruby в десятку общепризнанных языков
 программирования. Во многом, рост популярности языка приписывается
 популярности программного обеспечения, написанного на Ruby, в
-особенности – [Ruby on Rails][ror], среде разработки веб-приложений.
+особенности – [Ruby on Rails][ror], фреймворку для разработки веб-приложений.
 
 Ruby также [абсолютно открыт]({{ site.license.url }}). Открыт для бесплатного использования,
 изменений, копирования и распространения.
 
 ### Всё в Ruby – объекты.
 
 Вначале Matz рассматривал другие языки в поисках идеального синтаксиса.
-Вспоминая свои изыскания, он говорил: "Мне нужен был скриптовый язык,
+Вспоминая свои изыскания, он говорил: «Мне нужен был скриптовый язык,
 который был бы более мощным, чем Perl, и более объектно-ориентированным,
-чем Python."<sup>[2](#fn2)</sup>.”
+чем Python.»<sup>[2](#fn2)</sup>.”
```

Comment:
I think the `."` between "Python" and "\<sup>" was misplaced. See [the source in `en`](https://github.com/ruby/www.ruby-lang.org/blame/fa0136bcab5d6bd3662f8300762077c44fb1ec6e/en/about/index.md#L53).

```suggestion
чем Python<sup>[2](#fn2)</sup>.»
```